lms/templates/index.html

```diff
@@ -142,6 +154,26 @@ <h2 class="course-box__heading">${category.name}</h2>
         effect : "fadeIn"
       });
 
+      // category enrollment
+      $(function(){
+        $('.enroll-category').on('submit', function(event){
+          var slug = this.id;
+          event.preventDefault();
+          $.ajax({
+            url:'/enroll-category/',
+            type: "POST",
+            data: {'slug': slug},
+            success:function(response){
+              var $message=$("." + slug);
```

Comment:
Please add a more specific selector (e.g. include `enroll_status`).
It is for the future - in case we will have another element on the page with the same class but for a different purpose.

Reply:
Thanks. Fixed.